sentry-core/src/main/java/io/sentry/core/config/provider/JndiConfigurationProvider.java

```diff
@@ -0,0 +1,88 @@
+package io.sentry.core.config.provider;
+
+
+import io.sentry.core.ILogger;
+import io.sentry.core.SentryLevel;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.naming.NoInitialContextException;
```

Comment:
this cant live in the sentry-core package.
Android SDK has a subset of the Java SDK, javax is not one of them